operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -44,13 +38,11 @@ The following values need to be set in the boot set of the session template in o
 
 - `"rootfs_provider":` Set to `"sbps"`
 - To use the Node Management Network (NMN) for content projection,
-  - `"rootfs_provide_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-nmn._tcp.local:300"`
+  - `"rootfs_provide_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-nmn._tcp.iqn.2023-06.csm.iscsi:300"`
```

Comment:
Using the format `sbps:<schema version>:<IQN Domain>:<DNS SRV record reference>:<client discovery timeout in seconds>`, delimited by the colon `:` --

The first two fields are good. The 3rd field should be set to `iqn.2023-06.csm.iscsi` and is correct as well in this example. Note that the 3rd field is not designed in this case to be a valid DNS domain name, nor is it strictly required to by https://datatracker.ietf.org/doc/html/rfc3721#section-1.1. 

The 4th field, however, needs to be adjusted. It should be `_spbs-nmn._tcp.` or `_sbps-hsn._tcp.` followed by the installed systems domain name. This reference DOES have to be a valid SRV record reference that exists in DNS.

Reply:
I think my confusion stemmed from not understanding the difference between an IQN domain name and a system domain name. They both contain the phrase 'domain name', and I thought they were the same.

System domain name example: odin.hpc.amslabs.hpecorp.net
IQN domain name example: iqn.2023-06.csm.iscsi

I will go through the document again with this understanding in place.
However, I could still use edification. What contexts are these 'domain names' in? Are they in the same context? Is it all networking? I don't understand.